EN4_postprocessing/config.py

```diff
@@ -29,6 +29,7 @@ def __init__(self):
         self.grid_nc     = self.get_shell_var('GRID', True)
         self.dn_dat     = self.get_shell_var('DN_DAT', True)
         self.dn_out     = self.get_shell_var('DN_OUT', True)
+        self.case     = self.get_shell_var('MOD', True)
```

Comment:
**Non-actionable comments:**
. I couldn't spot where this new python variable was used.
. I'm not sure of the logic of creating a python variable with a different name to the shell variable. Maybe the shell variable needs to change?

Reply:
I use this for specifying the path. This replaces previous instances of 'P0.0', which meant that P0.0 data was being used regardless of what was set in the config.

Agreed. Happy to use either. We can change this in a future merge?